src/v/pandaproxy/schema_registry/storage.h

```diff
@@ -326,4 +326,29 @@ class schema_value_handler final : public json::base_handler<Encoding> {
     }
 };
 
+inline schema_value schema_value_from_iobuf(iobuf&& iobuf) {
+    auto p = iobuf_parser(std::move(iobuf));
+    auto str = p.read_string(p.bytes_left());
+    return json::rjson_parse(str.data(), schema_value_handler<>{});
+}
+
+inline iobuf schema_value_to_iobuf(
```

Comment:
would there be any added value hooking this up with the ADL api in reflection?

Reply:
> would there be any added value hooking this up with the ADL api in reflection?

I'm not sure yet. I haven't established what needs to be done for `schema_key::keytype`; currently `SCHEMA` is supported, but `CONFIG` is going to be supported soon, and that makes the serde more dynamic.